pgmpy/continuous/discretize/rounding.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import division
+
+import numpy as np
+
+from pgmpy.factors import ContinuousFactor
+from pgmpy.continuous.discretize import BaseDiscretizer
+
+
+class RoundingDiscretizer(BaseDiscretizer):
+    """
+    Class for discretization using the rounding method.
```

Comment:
@ankurankan I am rewording it to - 
This class uses the rounding method for discretizing the given continuous distribution.
Is it okay?